core/includes/form.inc

```diff
@@ -3756,10 +3794,33 @@ function theme_details($variables) {
     $element['#attributes']['open'] = 'open';
   }
 
+  $summary_content = '<span>' . $element['#summary'] . '</span>';
+
+  // Custom labels for the open/close toggle.
+  $more_less_toggle = '';
+  if (isset($element['#open_label']) && isset($element['#close_label'])) {
```

Comment:
I don't think the labels should be used to determine if the details has an open/close toggle.

Reply:
Fixed with [`f9b1a26` (#3629)](https://github.com/backdrop/backdrop/pull/3629/commits/f9b1a26b9dbc37ac2ae17b50ffdfd375cffb4771)